spec/controllers/dashboards_controller_spec.rb

```diff
@@ -83,25 +83,48 @@ def load_table(klass, options)
 
     it 'causes a CSV to be exported' do
       expect(Weam).to receive(:export)
-      get :export, csv_type: 'Weam', format: :csv
+      get :export, csv_type: 'Weam', format: :csv, number: 1
     end
 
     it 'includes filename parameter in content-disposition header' do
-      get :export, csv_type: 'Sva', format: :csv
+      get :export, csv_type: 'Sva', format: :csv, number: 1
       expect(response.headers['Content-Disposition']).to include('filename="Sva.csv"')
     end
 
-    it 'includes filename parameter in content-disposition header for institution' do
-      get :export, csv_type: 'Institution', format: :csv
-      expect(response.headers['Content-Disposition']).to include('filename="Institution.csv"')
-    end
-
     it 'redirects to index on error' do
       expect(get(:export, csv_type: 'BlahBlah', format: :csv)).to redirect_to(action: :index)
       expect(get(:export, csv_type: 'Weam', format: :xml)).to redirect_to(action: :index)
     end
   end
 
+  describe 'GET export_version' do
+    login_user
+
+    before(:each) do
+      defaults = YAML.load_file(Rails.root.join('config', 'csv_file_defaults.yml'))
+
+      InstitutionBuilder::TABLES.each do |klass|
+        load_table(klass, skip_lines: defaults[klass.name]['skip_lines'])
+      end
+
+      get :build
+    end
+
```

Comment:
Can we add a spec showing that an error is returned if you don't specify a `number` for an `institutions` export?

Reply:
`/dashboards/export/institutions/` would hit the route for the `export` method, which is already tested (it would be the same as someone exporting an invalid export type)

Reply:
I'm going to dismiss this because this is 10 days old and the institutions export route isn't accessible without a version number.